To: vim_dev@googlegroups.com
Subject: Patch 7.3.340
Fcc: outbox
From: Bram Moolenaar <Bram@moolenaar.net>
Mime-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit
------------

Patch 7.3.340
Problem:    When 'verbosefile' is set ftplugof.vim can give an error.
Solution:   Only remove filetypeplugin autocommands when they exist. (Yasuhiro
	    Matsumoto)
Files:	    runtime/ftplugof.vim


*** ../vim-7.3.339/runtime/ftplugof.vim	2010-08-15 21:57:11.000000000 +0200
--- runtime/ftplugof.vim	2011-10-20 16:51:00.000000000 +0200
***************
*** 1,11 ****
  " Vim support file to switch off loading plugins for file types
  "
  " Maintainer:	Bram Moolenaar <Bram@vim.org>
! " Last Change:	2002 Apr 04
  
  if exists("did_load_ftplugin")
    unlet did_load_ftplugin
  endif
  
! " Remove all autocommands in the filetypeplugin group
! silent! au! filetypeplugin *
--- 1,13 ----
  " Vim support file to switch off loading plugins for file types
  "
  " Maintainer:	Bram Moolenaar <Bram@vim.org>
! " Last Change:	2011 Oct 20
  
  if exists("did_load_ftplugin")
    unlet did_load_ftplugin
  endif
  
! " Remove all autocommands in the filetypeplugin group, if any exist.
! if exists("#filetypeplugin")
!   silent! au! filetypeplugin *
! endif
*** ../vim-7.3.339/src/version.c	2011-10-20 16:35:25.000000000 +0200
--- src/version.c	2011-10-20 18:01:18.000000000 +0200
***************
*** 711,712 ****
--- 711,714 ----
  {   /* Add new patch number below this line */
+ /**/
+     340,
  /**/

-- 
How many light bulbs does it take to change a person?

 /// Bram Moolenaar -- Bram@Moolenaar.net -- http://www.Moolenaar.net   \\\
///        sponsor Vim, vote for features -- http://www.Vim.org/sponsor/ \\\
\\\  an exciting new programming language -- http://www.Zimbu.org        ///
 \\\            help me help AIDS victims -- http://ICCF-Holland.org    ///
